File: generated_code/phi4/baseline_code_py/code_row_119.py
import random

# Generate three random floats
float1 = random.random()
float2 = random.random()
float3 = random.random()

# Convert them to strings with two decimal places using format (as an alternative to sprintf)
string = "{:.2f}".format(float1)
string2 = "{:.2f}".format(float2)
string3 = "{:.2f}".format(float3)

# Print the results
print(string, string2, string3)

File: generated_code/phi4/baseline_code_c/code_row_119.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    srand(time(NULL));

    float num1, num2, num3;
    char string[50], string2[50], string3[50];

    // Generate random floats between 0 and 1
    num1 = (float)rand() / RAND_MAX;
    num2 = (float)rand() / RAND_MAX;
    num3 = (float)rand() / RAND_MAX;

    // Convert to strings using sprintf
    sprintf(string, "%f", num1);
    sprintf(string2, "%f", num2);
    sprintf(string3, "%f", num3);

    printf("String: %s\n", string);
    printf("String2: %s\n", string2);
    printf("String3: %s\n", string3);

    return 0;
}